redaxo/src/addons/mediapool/lib/service_media.php

```diff
@@ -343,10 +343,10 @@ public static function getList(array $filter = [], array $orderBy = [], ?rex_pag
         }
 
         if ($pager) {
-            $query .= ' ORDER BY ' . implode(', ', $orderbys);
             $sql->setQuery(str_replace('SELECT m.filename', 'SELECT count(*)', $query), $queryParams);
             $pager->setRowCount((int) $sql->getValue('count(*)'));
 
+            $query .= '  ORDER BY ' . implode(', ', $orderbys);
```

Comment:
Mein Fehler, ein Leerzeichen zu viel ;) 

```suggestion
            $query .= ' ORDER BY ' . implode(', ', $orderbys);
```